app/src/views/team/[id]/CashRemunerationView.vue

```diff
@@ -1,10 +1,32 @@
 <template>
-  <div class="flex flex-col gap-y-4 py-6 lg:px-4 sm:px-6">
+  <div class="flex flex-col gap-6">
     <h2>CASH Remuneration</h2>
     <div class="flex gap-10">
-      <CashRemunerationCard cardType="balance" :amount="1000" />
-      <CashRemunerationCard cardType="month-claims" :amount="500" />
-      <CashRemunerationCard cardType="approved-claims" :amount="50" />
+      <OverviewCard
+        title="Total Balance"
+        bg-color="bg-[#C8FACD]"
+        :card-icon="bagIcon"
+        text-color="text-[#005249]"
+        currency="USD"
+        :amount="73900"
+        :previous-amount="52000"
+      />
```

Comment:
```suggestion
      <OverviewCard
        title="73.9K USD or 200 New Users"
        subtitle="Total Balance"
        :card-icon="bagIcon"
        variant="primary"
      >
      The content of the slot here
      So the dev can define what he want as a content here
    </OverviewCard>
```

The idea is to give more power to the parent.

Reply:
so we put the number on title? @hermannleboss . how about the loading?

Reply:
Yes, so for ex the parent format the data and send it as string